src/actionlist/action-list-tree.scss

```diff
@@ -0,0 +1,97 @@
+// <ul>
+// tree-view variant
+// renders ActionList with default styling for tree-view
+// collapse/expand icons presents as leadingActions, rotate 90deg
+// connecting vertical lines between collapse groups
+// consistent font-size between nested groups
+
+@mixin treeConnectingLine($left) {
+  position: absolute;
+  left: $left;
+  width: 1px;
+  height: 100%;
+  content: '';
+  background: var(--color-action-list-item-inline-divider);
+}
+
+:root {
```

Comment:
Anyone know how to get around this stylelint flag? I hesitate to ignore it- it seems important. This var is defined as an inline style, but I have an initial value set here 🤷 

`38:11  ✖  --ActionList-tree-depth is not defined  primer/no-undefined-vars`